### PR TITLE
Publish 2.10.0

### PR DESCRIPTION
**Pull request by victorbush**

### CODE REVIEW

File: ReleaseNotes.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 These are the NuGet package releases. See also [npm Release Notes](ReleaseNotesNpm.md).
 
+## 2.10.0
+
+* Add `file-name-suffix` generator setting.
```
Comment (victorbush) [inline ReleaseNotes.md:7]: There's been a number of commits since the package was published, so I'm not sure what all needs listed here.

Comment (ejball) [reply ReleaseNotes.md:7]: Hm, `Support generating Fastify plugins.` I guess. cc @mitchknife

Comment (mitchknife) [reply ReleaseNotes.md:7]: Oh good point. Yeah, that message works fine :+1:

Comment (mitchknife) [reply ReleaseNotes.md:7]: Also, sorry for not publishing a package earlier. That's bad form. 😁

**Pull request merged**